test/lsp/utils/work_done_progress.vimspec

```diff
@@ -0,0 +1,101 @@
+Describe lsp#internal#work_done_progress
+    Before
+        let g:lsp_work_done_progress_enabled = 1
+        call lsp#internal#work_done_progress#_enable()
+    End
+
+    After all
+        let g:lsp_work_done_progress_enabled = 0
+        call lsp#internal#work_done_progress#_disable()
+    End
+
+    It should be able to subscribe to $progress stream
+        call lsp#internal#work_done_progress#_disable()
```

Comment:
why not move this to `Before each` and `After each` so you don't need to do it every where.

Reply:
I fixed it.